enterprise/coderd/coderd.go

```diff
@@ -417,7 +417,7 @@ func (api *API) updateEntitlements(ctx context.Context) error {
 		if enabled {
 			var haCoordinator agpltailnet.Coordinator
 			if api.AGPL.Experiments.Enabled(codersdk.ExperimentTailnetPGCoordinator) {
-				haCoordinator, err = tailnet.NewPGCoord(ctx, api.Logger, api.Pubsub, api.Database)
+				haCoordinator, err = tailnet.NewPGCoord(api.ctx, api.Logger, api.Pubsub, api.Database)
```

Comment:
nit: oh, I thought that this is the same context passed down

Reply:
I _think_ so, but this is a much easier way to be sure.